apps/wasm/mvp/src/logger.rs

```diff
@@ -0,0 +1,79 @@
+// Copyright 2020 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0
+
+use log::{Level, Log, Metadata, Record};
+use wasm_bindgen::prelude::*;
+
+use crate::js_ffi::console_log_str;
+
+#[wasm_bindgen(inline_js = r###"
```

Comment:
Is the string based inline js code the only way to do this? or maybe move this to a js file with the module attribute if that works?

Reply:
Yeah we can do that but I just feel that makes it less readable since IDE does not provide code nav and it's a short code snippet. Does that make sense?

Reply:
got it, it should be fine.